rust/scx_loader/src/main.rs

```diff
@@ -139,11 +149,73 @@ impl ScxLoader {
     }
 }
 
+// Monitors CPU utilization and enables scx_lavd when utilization of any CPUs is > 90%
+async fn monitor_cpu_util() -> Result<()> {
+    let mut system = System::new_all();
+    let mut running_sched: Option<Child> = None;
+    let mut cpu_above_threshold_since: Option<Instant> = None;
+    let mut cpu_below_threshold_since: Option<Instant> = None;
+    
+    let high_utilization_threshold = 90.0;
+    let low_utilization_threshold_duration = Duration::from_secs(30);
+    let high_utilization_trigger_duration = Duration::from_secs(5);
```

Comment:
I'd be nice to have these as tunables in a config file (maybe `/etc/default/scx` or similar), but it's ok to have them hard-coded for now, so that we can test and see if we like the "automatic" mode.